lib/SILOptimizer/Mandatory/Differentiation.cpp

```diff
@@ -1198,10 +1213,32 @@ SILValue DifferentiationTransformer::promoteToDifferentiableFunction(
   auto *newDiffFn = context.createDifferentiableFunction(
       builder, loc, parameterIndices, resultIndices, origFnCopy,
       std::make_pair(derivativeFns[0], derivativeFns[1]));
-  context.addDifferentiableFunctionInstToWorklist(dfi);
+  context.getDifferentiableFunctionInstWorklist().push_back(dfi);
   return newDiffFn;
 }
 
+SILValue DifferentiationTransformer::promoteToLinearFunction(
+    LinearFunctionInst *lfi, SILBuilder &builder, SILLocation loc,
+    DifferentiationInvoker invoker) {
+  // TODO: Fill in. Copy code from above.
+  // For now, create a new `linear_function` instruction with an undef
+  // transpose.
+  // Eventually, use `emitTransposeFunctionReference` to fill in legitimately.
```

Comment:
Please update the explanation here:
```suggestion
  // Note: for now, this function creates a new `linear_function` instruction
  // with an undef transpose function operand. Eventually, a legitimate
  // transpose function operand should be created and used.
```